resources.py

```diff
@@ -0,0 +1,6 @@
+#
+#    This is a python library for resources like message templates etc.
+#
+
+class Message:
```

Comment:
It's best to always extend from object, ie:

Class Message(object):
  # ...

Old style classes are deprecated and I think don't even work on Python 2.7